### ISSUE TITLE
Expose CD name in multi-disc releases

### ISSUE THREAD

**Issue by nirvdrum** [beetbox/beets#3525]
### Use case

I have a special deluxe edition of Nirvana's "In Utero" album. While it's a multi-disc compilation, each disc contains a different recording of the album as is clearly labeled. Disc 1 is a remastered version of the original album, disc 2 is a remixed version of the album, and disc 3 is a live version of the album.

Unfortunately, the only way to get all three discs was as part of the deluxe edition, so Musicbrainz doesn't appear to have separate IDs for each disc. Instead, there's one for the whole compilation:

https://musicbrainz.org/release/dc02c14f-b3f2-40c2-a32b-4349f8d5a51f

Keeping track of each disc by number is something I'm definitely going to forget. And since each disc is essentially standalone, I'd like to be able to pick the album version I'd like to listen to at any given time. As a first step, I was looking to incorporate the disc name into the file path. From there, I was thinking of editing either the album name or the track titles to incorporate the disc name. Musicbrainz has the name of each disc available, but I don't see how I can access that from within beets.

### Solution

I think it would be helpful if beets made each disc's name available when processing a multi-disc compilation. Most of the time, the name is simple "1" or "2", but I have at least two multi-disc releases where each disc was given a name.

**Comment by sampsyo** [comment]
Hi! Can you try the $disctitle field? It should already contain this information.

**Comment by nirvdrum** [comment]
Thank you, @sampsyo. That indeed does what I'd like and I have no idea how I overlooked it. I spent a decent amount of time looking at `beet info` on a track and couldn't find anything that looked like the disc title. I see it there now, so I can only assume I overlooked it multiple times. I'm pleasantly surprised to see that it's empty in cases where not applicable, so I can put in in my path configuration without needing to add an inline rule to deal with the values.
